src/components/containers/UserResult/ResultSummary.js

```diff
@@ -0,0 +1,20 @@
+import { Component } from 'react';
+
+export default class ResultSummary extends Component {
+  render() {
+    const { profit, rateOfReturn } = this.props.result;
+
+    return (
+      <>
+        <p className="ResultSummary__profit">
+          <span className="ResultSummary__description">당첨 금액</span>
+          <span className="ResultSummary__value">{profit}원</span>
+        </p>
```

Comment:
저라면 요 단위를 컴포넌트화 해볼 것 같아요!
`Record`라는 이름으로요!
`ResultSummary`라는 컴포넌트가 재사용될까요? 🤔 

```suggestion
        <p className="record">
          <span className="record-label">{label}</span>
          <span className="record-value">{children}</span>
        </p>
```

Reply:
오어... 이거였습니다... 제가 하고싶었던게🤩 이렇게 누가봐도 재사용할 수 있을 것 같은 컴포넌트였는데... 현실은...🥲`ResultSummary`는 상위 컨테이너가 장황해져서 쪼개려는 목적으로 분리하게 되었습니다...! 위에 공유해주셨던 챠크라의 컴포넌트 천천히 살펴보고, 단순히 쪼개는 것보다 재사용성을 좀 더 고민하면서 이렇게 재사용 가능한 컴포넌트를 늘려보겠습니다!! 정말정말 시야가 트이는 느낌이에요! 감사합니다~🙇🏻‍♀️😆🙇🏻‍♀️

Reply:
단순 추출이 아닌, 의존성을 기반으로 추상화하는 연습을 해조시면 좋을 것 같아요! 높은 응집도 + 낮은 결합도!

어려운 말들이고 저도 계속 연습하고 있는 부분입니다 💪

Reply:
1단계 제한사항이 클래스 컴포넌트를 사용하는 것이라서 클래스 컴포넌트로 반영해보았습니다..!

📌 관련커밋: 61ff42a

Reply:
앗 코멘트를 이제봤네요...! `높은 응집도 + 낮은 결합도` 유념해서 도전해보겠습니다!! 🚀🚀🚀🚀 감사합니다! 💙